bundles/AdminBundle/Resources/public/js/pimcore/settings/fileexplorer/file.js

```diff
@@ -46,7 +46,9 @@ pimcore.settings.fileexplorer.file = Class.create({
 
             this.textarea = new Ext.form.TextArea({
                 value: response.content,
-                style: "font-family:courier"
+                fieldStyle: {
+                    "fontFamily": "courier"
```

Comment:
I guess it might be a matter of taste. Both are web safe (at least according to [MIT](https://web.mit.edu/jmorzins/www/fonts.html#monospace) and [w3schools](https://www.w3schools.com/cssref/css_websafe_fonts.asp)). I just opted against changing the font.

Maybe even something like `"Courier New, Courier, monospace"` or `"Lucida Console, Monaco, monospace"` could be a way to go. IMHO it wouldn't make a big difference as it's probably just a rarely used visual change.

Reply:
It is web-safe *if* you use Courier as a fall-back for Courier New. Courier alone does not work in Firefox on Ubuntu for example. Maybe just use `monospace` alone and let the browser decide?